zapgrpc/zapgrpc.go

```diff
@@ -21,7 +21,30 @@
 // Package zapgrpc provides a logger that is compatible with grpclog.
 package zapgrpc // import "go.uber.org/zap/zapgrpc"
 
-import "go.uber.org/zap"
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+const (
+	// See https://github.com/grpc/grpc-go/blob/v1.35.0/grpclog/loggerv2.go#L77-L86
+
+	grpcLvlInfo  = 0
+	grpcLvlWarn  = 1
+	grpcLvlError = 2
+	grpcLvlFatal = 3
+)
+
+var (
+	// grpc2zapLvl maps gRPC log levels to zap log levels.
+	// See https://pkg.go.dev/go.uber.org/zap@v1.16.0/zapcore#Level
+	grpc2zapLvl = map[int]zapcore.Level{
```

Comment:
nit: can we use camel case + full word here `_grpcToZapLevel` (since it's not used in a lot of places, better to keep it clear). we also prefixes our globals with `_` in zap

Reply:
Done